docker/flower.override.yml

```diff
@@ -0,0 +1,33 @@
+version: "3"
+
+services:
+  rabbitmq:
+    image: library/rabbitmq:3.8.19-management
```

Comment:
sorry If I am being annoying about this, could you please just change it to only the minor version? In this way it is the same as in the non-flower configuration